src/main/java/gregtech/common/pipelike/cable/tile/TileEntityCable.java

```diff
@@ -168,6 +168,7 @@ private void uninsulate() {
         world.setBlockState(pos, newBlock.getDefaultState());
         TileEntityCable newCable = (TileEntityCable) world.getTileEntity(pos);
         if (newCable != null) { // should never be null
+            // TODO: use transfer data method
```

Comment:
What's the status on this TODO?

Reply:
Will do that in a seperate pr after cover rewrite pr.